### ISSUE TITLE
recommend way of configuring Prometheus

### ISSUE THREAD

**Issue by pessoa** [timescale/tobs#72]
Once the stack is installed and running, what's the best way of configuring Prometheus? Adding targets, alert rules, alert manager receivers, etc.

**Comment by cevian** [comment]
@pessoa sorry for the late reply. On a high-level, you'd adjust these things using your value.yml using the 'prometheus' key and following the options in https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus/values.yaml. I'll let @blagojts give more details

@blagojts can we document this better in our values.yml?

**Comment by blagojts** [comment]
Yes, we need to update the values.yml file to point to the proper place now that helm hub has changed. The gist is that we're using the prometheus helm chart (https://artifacthub.io/packages/helm/prometheus-community/prometheus) as a dependency. All of the properties configurable in this chart are configured through the tobs chart the same way. Only difference:
1. Prometheus chart docs says to set the values for example like:
```
server:
  ingress:
    ## If true, Prometheus server Ingress will be created
    enabled: true
```
When using tobs, you need to prefix this with `prometheus
```
prometheus:
  ingress:
    enabled: true
```

As you can see here https://github.com/timescale/tobs/blob/master/chart/values.yaml#L65

For detailed info about what is configurable on the prometheus chart check out their docs (https://artifacthub.io/packages/helm/prometheus-community/prometheus) and more specifically the comments in their values file https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus/values.yaml

**Comment by pessoa** [comment]
Thanks! I'm at the point after
```
tobs install 
```

Where should I go next to customize the installed stack? Or that needs to be done before install?

**Comment by blagojts** [comment]
1. Most of the properties can be configured before installing (by specifying the appropriate values in the values.yaml file) Each of the services tobs installs exists as a separate helm chart which are combined in one umbrella chart that is tobs. You can check out the individual charts to see what they allow to be configured through the values file. The tobs helm chart sets up some default values so that the Promscale connector can connect to the db automatically, the db to create the appropriate user with specified password, for Prometheus tobs configures Promscale to be a remote_write endpoint endpoint, and for Grafana to have Prometheus and the Connector as data sources, and to have some default dashboards for monitoring the k8s cluster. 
2. After tobs is deployed, you can modify specific resources, for example the Prometheus helm chart deploys Prometheus in a way that the main prometheys.yml configuration is created as a ConfigMap which after it is modified you can restart the prometheus server instance and it will pick up the new config. Each of the different services deployed will have different ways to configure based on how it is deployed. For grafana for example, we enable persistance, persisting the config in timescaledb, everything you configure through the UI will be persisted in the db as config. You have to keep in mind that modifying the resources directly will make it harder to do upgrades with helm, since the changes will not be tracked by helm and thus overriden

**Comment by cevian** [comment]
Addressed by #75 #79
